src/Compilers/VisualBasic/Portable/Scanner/Scanner.vb

```diff
@@ -389,6 +389,18 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
 #End Region
 
 #Region "Buffer helpers"
+
+        Private Function NextAre(offset As Integer, chars As String) As Boolean
+            Debug.Assert(Not String.IsNullOrEmpty(chars))
+            Dim n = chars.Length
+            Dim i = -1
+            If CanGetCharAtOffset(offset + n) Then
+                Do
+                    i += 1
+                Loop While i < n AndAlso chars(i) = PeekAheadChar(offset + i)
+            End If
+            Return i = n
```

Comment:
It looks like you could just `Return False` if the condition `CanGetCharAtOffset(offset + n)` fails rather than evaluate `i = n`.